templates/aruba_os_show_arp.textfsm

```diff
@@ -0,0 +1,6 @@
+Value IP_ADDRESS (\d+.\d+.\d+.\d+)
+Value HARDWARE_ADDRESS (\w+.\w+.\w+.\w+.\w+.\w+)
+Value INTERFACE (\w+)
+
+Start
+ ^\s+Internet\s+${IP_ADDRESS}\s+${HARDWARE_ADDRESS}\s+${INTERFACE} -> Record
```

Comment:
We also want to track headers within the output to make sure we're capturing the correct data. If the headers were to ever change, we can easily create different state transitions to account for the changes in the data.
```suggestion
Start
 ^IPV4\s+ARP\s+Table
 ^-+
 ^Protocol\s+IP\s+Address\s+Hardware\s+Address\s+Interface\s+Age
 ^\s+Internet\s+${IP_ADDRESS}\s+${HARDWARE_ADDRESS}\s+${INTERFACE} -> Record
 ^\s*$$
 ^. -> Error
```